aries_cloudagent/core/conductor.py

```diff
@@ -144,7 +144,6 @@ async def setup(self):
                     self.root_profile.BACKEND_NAME == "askar"
                     and ledger.BACKEND_NAME == "indy-vdr"
                 ):
-                    context.injector.bind_instance(BaseLedger, ledger)
                     context.injector.bind_provider(
```

Comment:
For me, it is working even with `context.injector.bind_instance(BaseLedger, ledger)` included.